### ISSUE TITLE
[BUG]the transactions are not the same with blance.

### ISSUE THREAD

**Issue by Firesuiry** [Chia-Network/chia-blockchain#8612]
![image](https://user-images.githubusercontent.com/30189673/134906855-aa17e816-1bf2-470f-b68a-e12bd6f4bf14.png)
that is the transaction record.
![image](https://user-images.githubusercontent.com/30189673/134906943-96cead6e-201e-415f-a8cf-9cd6185dceac.png)
this is the blance.

my two Outgoing transcations disappear and two Incoming transcations appear.

**Comment by Firesuiry** [comment]
if the transcations record is right, I should have about 0.04xch.
but now i have about 0.02xch.

**Comment by Firesuiry** [comment]
> i guess you created plotNFT ? the plotNFT creation does not appear in the wallet, but the change you receive back from the transaction does show up as incoming, and that would give the impression of more XCH should be in the wallet... it is correctly showing what is in the wallet though 0.02XCH~

I have not created plotNFT. The key is a new key I created to test. I have transfer 0.00999 xch 2 times into this key and transfer 10 mojos 5 times out of this key. but the first two transfer out display as income in block 916168, this really confuse me.

**Comment by loppefaaret** [comment]
have you resynced the wallet database after those transactions out ?

**Comment by Firesuiry** [comment]
> have you resynced the wallet database after those transactions out ?

I have synced the wallet database after those transactions out 
and 
is your meaning deleting wallet database and then resyncing? if so, I have not.

I have saw the chia explorer website and there are two income transcation record too.

I guess my first two 0.00999xch was divided into two 0.0009989...90xch and two 10 mojos, the two 10 mojos to pay the outgoing and the 0.0009989...90xch display as income. 

but if so, the problem is there should be tow outgoing transcation to make my tow 0.00999xch disappear

**Comment by loppefaaret** [comment]
ok, you did not resync the wallet database after those 2 transaction that are missing - did you do them from another client than the one showing in the screenshot then ?

**Comment by Firesuiry** [comment]
No, I have another client but it sync very slow,  after 24  hours it only sync 200K blocks... 
I guess it will be synced after 4 days.

**Comment by Firesuiry** [comment]
> ok, you did not resync the wallet database after those 2 transaction that are missing - did you do them from another client than the one showing in the screenshot then ?

![image](https://user-images.githubusercontent.com/30189673/136220895-db65f80c-ab7d-4191-b83a-dd02ded10524.png)
